README.md

```diff
@@ -219,15 +219,29 @@ Now you are all set to run
 make install
 ```
 
-## Build
+## Build the vagrant box
```

Comment:
Remember to update the `Content` section `3. [Build](#build)` :point_right: `3. [Build the vagrant box](#build-the-vagrant-box)`

Reply:
Thanks!